src/main/play-doc/operation/ClusterConfiguration.md

```diff
@@ -0,0 +1,180 @@
+
+### ConductR Agent service user
+
+The `conductr-agent` service runs as the daemon user `conductr-agent` in the user group `conductr-agent`. When the service is started the first time it creates the user and group itself.
+
+The `conductr-agent` user executes the commands in the background without any shell. You can specify additional environment variables in `/etc/default/conductr-agent`. This file will be sourced before the actual service gets started.
+
+### Configuring ConductR Core
+
+The configuration file `/usr/share/conductr/conf/conductr.ini` is the primary configuration file for the ConductR Core service. This file is used to specify Core ConductR service settings, such as `-Dconductr.ip` used during installation. See the comments section of the `conductr.ini` file for more examples.
+
+The ConductR Core service must be restarted after changes to the `conductr.ini` to take effect.
+
+### Configuring ConductR Agent
+
+The configuration file `/usr/share/conductr-agent/conf/conductr-agent.ini` is the primary configuration file for the ConductR Agent service. This file is used to specify ConductR Agent service settings, such as `-Dconductr.agent.roles` used during installation. See the comments section of the `conductr-agent.ini` file for more examples.
+
+Akka module configuration can also be set using this file. For example, to assign a ConductR node the roles of `megaIOPS` and `muchMem` instead of the default, `web`, set `akka.cluster.roles` in `conductr-agent.ini`:
+
+```bash
+echo -Dconductr.agent.roles.0=megaIOPS | sudo tee -a /usr/share/conductr-agent/conf/conductr-agent.ini
+echo -Dconductr.agent.roles.1=GPU | sudo tee -a /usr/share/conductr-agent/conf/conductr-agent.ini
+sudo service conductr-agent restart
+```
+
+With this setting, the node would offer the roles `megaIOPS` and `GPU`. Only bundles with a `BundleKeys.roles` of `megaIOPS,` `GPU` or both `megaIOPS` and `GPU` will be loaded and run on this node.
+
+The ConductR Agent service must be restarted after changes to the `conductr-agent.ini` to take effect. Role matching must also be enabled on the cluster core nodes for bundles to be scheduled by role.
+
+### Roles
+
+Roles allow machines to be targeted for specific purposes. Some machines may have greater IO capabilities than others, some may have more CPU, memory, and other resources. Some may also be required to maintain a volume that holds a database.
+
+When getting started with ConductR, it is reasonable to start with ConductR not considering roles during scheduling. ConductR has the default settings `-Dconductr.resource-provider.match-offer-roles=off`.
+
+However, when moving into a production scenario, you should plan and assign roles for your ConductR cluster. The roles can be enabled by modifying the settings `-Dconductr.resource-provider.match-offer-roles=on` as such.
```

Comment:
Note sure what you mean by that. Note that this section is not changed by the PR. This is how it currently documented.